examples/prometheus/prometheus.yaml

```diff
@@ -291,3 +301,196 @@ objects:
+  kind: Deployment
+  metadata:
+    labels:
+      app: alerts
+    name: alerts
+    namespace: "${NAMESPACE}"
+  spec:
+    replicas: 1
+    selector:
+      matchLabels:
+        app: alerts
+    template:
+      metadata:
+        labels:
+          app: alerts
+        name: alerts
+      spec:
+        serviceAccountName: prometheus
+        containers:
+        - name: oauth-proxy
+          image: ${IMAGE_PROXY}
+          imagePullPolicy: IfNotPresent
+          ports:
+          - containerPort: 8443
+            name: web
+          args:
+          - -https-address=:8443
+          - -email-domain=*
+          - -client-id=system:serviceaccount:${NAMESPACE}:prometheus
+          - -upstream=http://localhost:9099
+          - -provider=openshift
+          - -redirect-url=https:///oauth2/callback
+          - '-openshift-sar={"namespace": "${NAMESPACE}", "verb": "list", "resource": "services"}'
+          - -tls-cert=/etc/tls/private/tls.crt
+          - -tls-key=/etc/tls/private/tls.key
+          - -client-secret-file=/var/run/secrets/kubernetes.io/serviceaccount/token
+          - -cookie-secret-file=/etc/proxy/secrets/session_secret
+          - '-openshift-delegate-urls={"/": {"resource": "namespaces", "verb": "get", "name": "${NAMESPACE}"}}'
+          volumeMounts:
+          - mountPath: /etc/tls/private
+            name: alerts-tls
+          - mountPath: /etc/proxy/secrets
+            name: secrets
+
+        - name: message-buffer
+          image: ${IMAGE_MESSAGE_BUFFER}
+          imagePullPolicy: IfNotPresent
+          volumeMounts:
+          - mountPath: /message-buffer
+            name: message-buffer-data
+          ports:
+          - containerPort: 9099
+            name: message-buf
+
+        - name: alertmanager
+          args:
+          - -config.file=/etc/alertmanager/alertmanager.yml
+          image: ${IMAGE_ALERTMANAGER}
+          imagePullPolicy: IfNotPresent
+          ports:
+          - containerPort: 9093
+            name: web
+          volumeMounts:
+          - mountPath: /etc/alertmanager
+            name: alertmanager-config
+          - mountPath: /alertmanager
+            name: alertmanager-data
+
+        restartPolicy: Always
+        volumes:
+        - configMap:
+            defaultMode: 420
+            name: alertmanager
+          name: alertmanager-config
+        - name: secrets
+          secret:
+            secretName: alerts-proxy
+        - name: alerts-tls
+          secret:
+            secretName: alerts-tls
+        - emptyDir: {}
+          name: alertmanager-data
+        - emptyDir: {}
+          name: message-buffer-data #TODO: make persistent
+- apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    name: alertmanager
+    namespace: "${NAMESPACE}"
+  data:
+    alertmanager.yml: |
+      global:
+
+      # The root route on which each incoming alert enters.
+      route:
+        # default route if none match
+        receiver: message-buffer-wh
```

Comment:
We need to have this root receiver, with a very prominent comment saying not to change it
(because manageiq won't work) but change hid child.
We also need to add a child route so that users will have a course of action for alerts.

This is an origin example so I don't know if you want to do it here or somewhere else.
If this is general example you might want to change this with something more generic(e.g smtp).